pkg/controllers/jobflow/state/running.go

```diff
@@ -28,6 +28,8 @@ func (p *runningState) Execute(action v1alpha1.Action) error {
 		return SyncJobFlow(p.jobFlow, func(status *v1alpha1.JobFlowStatus, allJobList int) {
 			if len(status.CompletedJobs) == allJobList {
 				status.State.Phase = v1alpha1.Succeed
+			} else if len(status.FailedJobs) > 0 {
```

Comment:
If there is a job's state failed, then the jobflow's state is failed? It feels too extreme

Reply:
vcjob faild is the final state after multiple retries. Its failure should be reflected in the final state of the jobflow.
Currently, vcjob faild will hinder the execution of dependent vcjobs and will remain in the running state for a long time.

Reply:
If we implement `if` logic for jobflow later. The failed vcjob can also trigger the next vcjob to run.

if complete:
   run job A
else if failed:
   run job B

Reply:
Perhaps we can use the status of the leaf node to represent the status of the jobflow

![image](https://github.com/user-attachments/assets/912c4941-e58f-49ab-b9b8-8e64d15d2d5e)

Reply:
![image](https://github.com/user-attachments/assets/b3cd665d-7f9b-4241-9989-444234c68521)

Reply:
Great. 
First, `jobflow.Spec.Flow` needs to belong to DAG

When we add conditional judgment, we can add `IsConditionFail(status.FailedJobs, jobs.Depends) bool` about：Check the leaf Node(Make sure there is a Failed status processing branch)